mobile-app/app/screens/AppNavigator/screens/Balances/components/BottomSheetAssetSortList.tsx

```diff
@@ -0,0 +1,81 @@
+import { memo } from 'react'
+import { ThemedFlatList, ThemedIcon, ThemedText, ThemedView, ThemedTouchableOpacity } from '@components/themed'
+import { BottomSheetFlatList } from '@gorhom/bottom-sheet'
+import { Platform, TouchableOpacity } from 'react-native'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import { useThemeContext } from '@shared-contexts/ThemeProvider'
+
+export interface BottomSheetAssetSortProps {
+  headerLabel: string
+  onCloseButtonPress: () => void
+  onButtonPress: (item: string) => void
+  modifiedDenominationCurrency: string
+}
+
+export const BottomSheetAssetSortList = ({
+  headerLabel,
+  onCloseButtonPress,
+  onButtonPress,
+  modifiedDenominationCurrency
+}: BottomSheetAssetSortProps): React.MemoExoticComponent<() => JSX.Element> => memo(() => {
+  const { isLight } = useThemeContext()
+  const flatListComponents = {
+    mobile: BottomSheetFlatList,
+    web: ThemedFlatList
+  }
+  const FlatList = Platform.OS === 'web' ? flatListComponents.web : flatListComponents.mobile
+  const assetSortList: string[] = [`Highest ${modifiedDenominationCurrency} value`, `Lowest ${modifiedDenominationCurrency} value`, 'Highest token amount', 'Lowest token amount', 'A to Z', 'Z to A']
+
+  const renderItem = ({
+    item,
+    index
+  }: {
+    item: string
+    index: number
+  }): JSX.Element => (
+    <ThemedTouchableOpacity
+      style={tailwind('px-4 py-3 ')}
+    >
+      <ThemedText
+        testID={`select_asset_${item}`}
```

Comment:
Open sort dropdown -> Choose which sorting (sometimes doesn't work at the first time)

One minor UX improvement: Sorting should work even when you're not pressing right at the center of each row